src/main/java/ch/njol/skript/util/Timespan.java

```diff
@@ -131,118 +173,112 @@ public static Timespan parse(final String s) {
 				
 			}
 		}
+
 		return new Timespan(t);
 	}
 	
-	private final long millis;
-	
 	public Timespan() {
 		millis = 0;
 	}
 	
-	public Timespan(final long millis) {
+	public Timespan(long millis) {
 		if (millis < 0)
 			throw new IllegalArgumentException("millis must be >= 0");
 		this.millis = millis;
 	}
 	
 	/**
 	 * @deprecated Use fromTicks_i(long ticks) instead. Since this method limits timespan to 50 * Integer.MAX_VALUE.
-	 * @addon I only keep this to allow for older addons to still work. / Mirre
+	 * Keeping this for older addons to stay working.
 	 */
 	@Deprecated
-	public static Timespan fromTicks(final int ticks) {
-		return new Timespan(ticks * 50L);
+	public static Timespan fromTicks(int ticks) {
+		return new Timespan(ticks * TimePeriod.TICK.time);
 	}
 	
-	public static Timespan fromTicks_i(final long ticks) {
-		return new Timespan(ticks * 50L);
+	public static Timespan fromTicks_i(long ticks) {
+		return new Timespan(ticks * TimePeriod.TICK.time);
 	}
-	
+
 	public long getMilliSeconds() {
```

Comment:
I wonder if it's worth adding a default `TimePeriod.MILLISECOND` option

Reply:
Not sure about that, It's just a `1L` which doesn't have much use in math/comparison otherwise let me know :)